tests/integration/targets/mso_schema_site_anp_epg_selector/tasks/main.yml

```diff
@@ -0,0 +1,1017 @@
+  - {selector: 'site_selector_2'}
+  - {selector: 'site_selector_3'}
+  register: nm_add_site_selectors_again
+
+- name: Verify nm_add_site_selectors_again
+  assert:
+    that:
+    - nm_add_site_selectors_again is changed
+
+# Query all selectors
+- name: Query selectors to site EPG
+  mso_schema_site_anp_epg_selector:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: 'aws_{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    anp: ANP
+    epg: ansible_test_1
+    state: query
+  register: query_all_site_selectors
+
+- name: Verify query_all_site_selectors
+  assert:
+    that:
+    - query_all_site_selectors is not changed
+    - query_all_site_selectors.current | length == 3
+    - query_all_site_selectors.current[0].name == "site_selector_1"
+    - query_all_site_selectors.current[0].expressions[0].key == "Custom:expression_1"
+    - query_all_site_selectors.current[0].expressions[0].operator == "in"
+    - query_all_site_selectors.current[0].expressions[0].value == "test"
+    - query_all_site_selectors.current[1].name == "site_selector_2"
+    - query_all_site_selectors.current[1].expressions[0].key == "Custom:expression_1"
+    - query_all_site_selectors.current[1].expressions[0].operator == "in"
+    - query_all_site_selectors.current[1].expressions[0].value == "test"
+    - query_all_site_selectors.current[2].name == "site_selector_3"
+    - query_all_site_selectors.current[2].expressions[0].key == "Custom:expression_1"
+    - query_all_site_selectors.current[2].expressions[0].operator == "in"
+    - query_all_site_selectors.current[2].expressions[0].value == "test"
+
+# Query sepecific seletor to site EPG
+- name: Query selector to site EPG
+  mso_schema_site_anp_epg_selector:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: 'aws_{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    anp: ANP
+    epg: ansible_test_1
+    selector: site_selector_1
+    state: query
+  register: query_site_selector_1
+
+- name: Verify query_site_selector_1
+  assert:
+    that:
+    - query_site_selector_1 is not changed
+    - query_site_selector_1.current.name == "site_selector_1"
+    - query_site_selector_1.current.expressions[0].key == "Custom:expression_1"
+    - query_site_selector_1.current.expressions[0].operator == "in"
+    - query_site_selector_1.current.expressions[0].value == "test"
+
+- name: Remove site selector 3 (normal_mode)
+  mso_schema_site_anp_epg_selector:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: 'aws_{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    anp: ANP
+    epg: ansible_test_1
+    selector: site_selector_3
+    state: absent
+  register: nm_remove_site_selector_3
+
+- name: Verify nm_remove_site_selector_3
+  assert:
+    that:
+    - nm_remove_site_selector_3 is changed
+    - nm_remove_site_selector_3.current == {}
+
```

Comment:
Missing a Remove site selector 3 again (normal mode) so that you trigger the last missing line for an absent of an already absent selector.

So you can mostly copy line 625 to 641 and just change the "is changed" into "is not changed" (and rename the variable)